packages/frp/build.sh

```diff
@@ -1,4 +1,4 @@
-TERMUX_PKG_HOMEPAGE=https://gofrp.org/doc
+TERMUX_PKG_HOMEPAGE=https://gofrp.org/
```

Comment:
Consider using either the English language page (equivalent to the previous `/doc` page), or GitHub repo instead.
```suggestion
TERMUX_PKG_HOMEPAGE=https://gofrp.org/en
```
See: https://repology.org/project/frp/information for more info.